package.json

```diff
@@ -80,8 +81,8 @@
     "react-axe": "^3.1.0",
     "react-test-renderer": "^16.2.0",
     "redux-mock-store": "^1.3.0",
-    "stylelint": "^8.4.0",
-    "stylelint-scss": "^2.2.0",
+    "stylelint": "^10.0.1",
+    "stylelint-scss": "^3.6.1",
```

Comment:
`stylelint-scss` is a dependency of `stylelint-config-wagtail`, so can be removed from here.